com.github.rhyme_player.rhyme.yml

```diff
@@ -0,0 +1,83 @@
+app-id: com.github.rhyme_player.rhyme
+runtime: org.freedesktop.Platform
+runtime-version: "20.08"
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "20.08"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node14
+command: run.sh
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=home
+build-options:
+  append-path: /usr/lib/sdk/node14/bin
+  cflags: -O2 -g
+  cxxflags: -O2 -g
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+modules:
+  - name: yarn
```

Comment:
Yarn was added to the nodejs SDK extension, you don't need to build it yourself.

Reply:
do I need to set in environment variables for that

Reply:
No, just remove the `yarn` module.